src/components/Loading/Loading.styles.js

```diff
@@ -0,0 +1,30 @@
+import { css, keyframes } from '@emotion/core';
+import { Radius, Space } from 'lib/theme';
+
+const spin = keyframes({
+  to: { transform: 'rotate(360deg)' },
+});
+
+const styles = {
+  loader: css({
+    width: 30,
+    height: 30,
```

Comment:
Dumb thing, but can you put this value in a constant in this file? `const loaderSize = 30`??